client/src/store/modules/scheduledEvent/normalizer.js

```diff
@@ -30,15 +30,14 @@ export const eventEmailMapper = Email => ({
 
 export const eventApiMapper = Event => ({
     id: Event.id,
+    status: Event.status,
     invitee_name: Event.name,
     invitee_email: Event.email,
     invitee_question: Event.question,
-    status: Event.status,
     start_date: moment(Event.startDate)
         .tz(Event.timezone)
         .utc()
         .format('YYYY-MM-DD HH:mm:ss'),
     location: Event.location,
-    timezone: Event.timezone,
-    event_type: eventTypeMapper(Event.eventType)
```

Comment:
Why you delete Event type? I need this data in events list

Reply:
I have an error with it.

Reply:
Which error do you have?

Reply:
When I am trying to cancel event, I receive "red" console with "'status' of undefined...".
But I've found where mistake is.
Just need to rename `tags` property in mappers to `tagChecks`, as said @yevhen-turovets, seems to be truth